config/sync/s3fs.settings.yml

```diff
@@ -3,7 +3,7 @@ _core:
 credentials_file: ''
 bucket: ''
 region: us-gov-west-1
-use_cname: true
+use_cname: false
```

Comment:
We probably don't mean to change this file.

Reply:
Just FYI, I'm going through this and looking for files appear to be changing by accident, or that we committed but are no longer using. I'm using the "review" to mark things so I can easily find them and correct them in a bit.